bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/controller/search/BahmniPatientSearchController.java

```diff
@@ -44,7 +44,7 @@ public BahmniPatientSearchController(BahmniPatientService bahmniPatientService)
         RequestContext requestContext = RestUtil.getRequestContext(request, response);
         PatientSearchParameters searchParameters = new PatientSearchParameters(requestContext);
         try {
-            List<PatientResponse> patients = bahmniPatientService.search(searchParameters);
```

Comment:
After making this change, are there any remaining usages of bahmniPatientService.search?